spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -145,11 +145,11 @@ public void testDeleteFileThenMetadataDelete() throws Exception {
 
     // Metadata Delete
     Table table = Spark3Util.loadIcebergTable(spark, tableName);
-    Set<DataFile> dataFilesBefore = TestHelpers.dataFiles(table);
+    Set<DataFile> dataFilesBefore = TestHelpers.uniqueDataFiles(table);
```

Comment:
Does the test pass with using dataFiles?  (if we change to List)  Looks like we are just comparing size, which should hopefully work?

Just wanted to avoid another test method if possible.

Reply:
@szehon-ho Got it.

As long as you don't need uniqueness, I can pick List-based approach and test again.

Reply:
@szehon-ho `List` also works (tested locally).